### ISSUE TITLE
Spring Modulith verifications broken

### ISSUE THREAD

**Issue by odrotbohm** [spring-projects/sts4#1127]
If you import Spring RESTBucks into VSCode the Spring Modulith verification rejects code in `OrderInitializer` looking as follows:

<img width="1312" alt="Screenshot 2023-10-19 at 23 07 35" src="https://github.com/spring-projects/sts4/assets/128577/18be4a06-dae1-4bdb-868f-6add62d5e37d">

All types are marked with the message “Type not exposed by core module” despite the types not residing in the core module.

**Comment by BoykoAlex** [comment]
I have attempted not to resolve anything while reconciling but it looks like i just broke everything for the case Modulith. Hopefully, I have corrected most of the things with 58635fda1ccca5b49b96a7fd49a81761ec34ec9b

Currently, I only see one issue: `org.springsource.restbucks.order.web.LocationAndDrinks` is considered to be from `order` module... I suppose that the only case when `LocationAndDrinks` would belong to `order` module is if the package was ``org.springsource.restbucks.order.internal` rather than `org.springsource.restbucks.order.web`. Is this correct? Thus `internal` is a special case where i can assume that parent package is the module, right?
